.gemspec

```diff
@@ -38,5 +38,45 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency(%q<hiera>, [">= 2.0", "< 4"])
   # PUP-7115 - return to a gem dependency in Puppet 5
   # s.add_runtime_dependency(%q<semantic_puppet>, ['>= 0.1.3', '< 2'])
+  # i18n support (gettext-setup and dependencies)
   s.add_runtime_dependency(%q<gettext-setup>, [">= 0.10", "< 1"])
+  s.add_runtime_dependency(%q<locale>, "~> 2.1")
+  # Hiera has an unbound dependency on json_pure
+  # json_pure 2.0.2+ officially requires Ruby >= 2.0, but should have specified that in 2.0
+  s.add_runtime_dependency(%q<json_pure>, "~> 1.8")
+  # net-ssh is a runtime dependency of Puppet::Util::NetworkDevice::Transport::Ssh
+  s.add_runtime_dependency(%q<net-ssh>, "~> 2.1")
+
+  # development dependencies
+  s.add_development_dependency(%q<rake>, "10.1.1")
```

Comment:
The plan wasn't necessarily to remove in a future commit - it was to try and move all the development dependencies into the `.gemspec`.

But I'll remove it from here for now and it can be addressed later in PUP-7433